content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,115 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+
+
+Since the dawn of DJing, spinning vinyl records never really went out of fashion in the DJ community.
+Even when CDs became popular in 90s and most music listeners happily phased out their record players, the classical "two turntables and a mixer" setup stayed the epitome of DJing.
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the tactile feeling of manipulating music with your hands is still very different from what most controllers can do.
```

Comment:
"can do" is a little awkward here -- the key phrase is "the tactile feeling", and a feeling is not something that a controller "does".  How about: "the tactile feeling of manipulating music with your hands is something controllers have had trouble achieving."